dev/com.ibm.ws.security.spnego_fat/fat/src/com/ibm/ws/security/spnego/fat/FATSuite.java

```diff
@@ -138,25 +128,6 @@ private boolean isHybridJDK(JavaInfo javaInfo) {
         }
     };
 
-    /**
```

Comment:
Do you plan to support running against the old KDCs? If so, this might be an issue if the setup rule fails. It will leave artifacts.

Reply:
We will for some of the testing scenarios that we are unable to use the local ApacheDS KDC for. For those cases I plan to consolidate all of them to the spnego_fat.1 project. that way the fatsuite and rules will be setup specifically for handling the fyre KDC's and cleanup